app.js

```diff
@@ -1,8 +1,112 @@
 // import functions and grab DOM elements
-
+const currentPollEl = document.getElementById('current-poll-container');
+const newPollEl = document.getElementById('new-poll');
+const optionAAddButton = document.getElementById('option-a-add-button');
+const optionASubtractButton = document.getElementById('option-a-sub-button');
+const optionBAddButton = document.getElementById('option-b-add-button');
+const optionBSubtractButton = document.getElementById('option-b-sub-button');
+const finishPollButton = document.getElementById('finish-poll-button');
+const pastPollsContainer = document.getElementById('past-poll-container');
 // let state
+let question = '';
+let optionA = '';
+let optionB = '';
+let votesA = 0;
+let votesB = 0;
 
+let pastPolls = [];
 // set event listeners 
+newPollEl.addEventListener('submit', (e) => {
+    e.preventDefault();
+    const data = new FormData(newPollEl);
+    const userQuestion = data.get('question');
+    const userOptionA = data.get('questionA');
+    const userOptionB = data.get('questionB');
+    question = userQuestion;
+    optionA = userOptionA;
+    optionB = userOptionB;
+
+
+    displayCurrentPoll();
+
+});
+
+optionAAddButton.addEventListener('click', () => {
+    votesA++;
+    console.log('votesA');
+    displayCurrentPoll();
+});
+optionBAddButton.addEventListener('click', () => {
+    votesB++;
+    displayCurrentPoll();
+});
+
+optionASubtractButton.addEventListener('click', () => {
+    votesA--;
+    displayCurrentPoll();
+});
+
+optionBSubtractButton.addEventListener('click', () => {
+    votesB--;
+    displayCurrentPoll();
+});
+
+finishPollButton.addEventListener('click', () => {
+    pastPollsContainer.textContent = '';
```

Comment:
Since this is something that will happen every time you call `displayPastPolls()` it might be worth just moving this line inside that function definition so then you don't have to worry about remembering to do this each time you invoke the function.